node/src/blockchain/blockchain_bridge.rs

```diff
@@ -565,7 +459,7 @@ impl BlockchainBridge {
             if start_block_number == u64::MAX {
                 start_block_number
             } else {
-                start_block_number + 1u64 // TODO: GH-744 Way are we adding +1 can we just return the same value?
+                start_block_number + 1u64
```

Comment:
Yeah, I believe this is wrong. Careful here. But if you agree with me after I have explained you, do the cleanup.